compose/src/main/java/com/bottlerocketstudios/compose/util/BinaryExtUtil.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.compose.util
+
+import android.graphics.BitmapFactory
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+
+fun ByteArray.convertToImageBitmap(): ImageBitmap? {
+    return try {
+        BitmapFactory.decodeByteArray(this, 0, this.count()).asImageBitmap()
+    } catch (e: Exception) {
+        null
```

Comment:
@VanyaHuamanBottleRocket Let's log the exception with Timber to aid in future debugging/error understanding